### PR TITLE
Cart wishlist management

### PR DESCRIPTION
**Pull request by swethaogeti**

### CODE REVIEW

**Comment by swethaogeti** [comment]
for preview-https://shiny-brioche-240be3.netlify.app/

File: src/components/index.js [hunk 1]
```diff
@@ -6,4 +6,7 @@ import Footer from "./footer/Footer";
 import Brands from './brands/Brands';
 import ProductCard from "./productCard/ProductCard";
```
Comment (theVivek-kumar) [inline src/components/index.js:7]: don't mix name export with default export

File: src/components/productCard/ProductCard.js [hunk 2]
```diff
@@ -12,14 +18,27 @@ function ProductCard({image,brand,title,rating,id,price,discount,total_price,tot
    </div>
    
    
-   <div className='badge text-badge icon-badge'><i className='far fa-heart' ></i></div>
+   <div className='badge text-badge icon-badge'>
+   
+   {wishlist.some((c)=>c.id===id)?( <i className='fa fa-heart' style={{color:'#8F00FF'}} onClick={()=>{cartDispatch({type:'REMOVE_FROM_WISHLIST',item:id})}}></i>):
+
+   <i className='far fa-heart'  onClick={()=>{cartDispatch({type:'ADD_TO_WISHLIST',item:{image,brand,title,id,price,discount,total_price,amount}})}}></i>
```
Comment (theVivek-kumar) [inline src/components/productCard/ProductCard.js:25]: use card component to map the products

**Review by theVivek-kumar — APPROVED** [review]

File: src/components/wishlistCard/WishlistProduct.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <div className='wishlist-prd product-card'>
+    <div className="card-container flex-box-default">
+    <div className="badge-container">
+    <img src={image} className="image-responsive product-img"/>
+                        
+ </div>
+ 
+ {cart.some((i)=>i.id===id)
+    ?(<div className='badge text-badge icon-badge active-icon'><i className='fas fa-shopping-cart' ></i></div>)
+
```
Comment (theVivek-kumar) [inline src/components/wishlistCard/WishlistProduct.js:18]: use use effect for wish list and product page

**Review by theVivek-kumar — APPROVED** [review]

File: src/components/cartCard/CartProduct.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+><i className='fa fa-minus'></i></button>
+<span>{amount}</span>
+<button onClick={()=>{cartDispatch({type:'INCREASE',item:id})}} 
+
+><i className='fa fa-plus'></i></button>
+</div>
+<div className="hz-btn-container flex-box-default">
+<button className="btn btn-primary"
+onClick={()=>{cartDispatch({type:'REMOVE_FROM_CART', item:id})}}
+>DELETE</button>
```
Comment (theVivek-kumar) [inline src/components/cartCard/CartProduct.js:38]: your website is breaking use useEffect  and UI is good to use box shadow  for card component  implement rating component and short by

**Review by BISINIGISHIVANI — APPROVED** [review]
functionalities working fine.ui was good.

**Pull request merged**